### ISSUE TITLE
Vaadin 8 Window Keyboard Maximize Accessibility

### ISSUE THREAD

**Issue by dyachera** [vaadin/framework#11517]
This is in relation to post: https://vaadin.com/forum/thread/17584294/vaadin-8-window-keyboard-accessability

I am currently working with Vaadin 8.7.1. 

I am attempting to make my application 508 compliant. I noticed that in the Window component you can successfully tab to the maximize button in the window to change its state, however trying to activate it with the keyboard (space/enter) does not work, only clicking with mouse triggers it. I believe since it does contain other accessibility features such as tabindex and aria-label that it should also be able to be activated by the keyboard.  I also do not know of anyway I would be able to grab a reference to it to create my own shortcut listener.
